pkg/asset/manifests/aws/cluster.go

```diff
@@ -151,15 +151,45 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 				PresignedURLDuration: &metav1.Duration{Duration: 1 * time.Hour},
 			},
 			ControlPlaneLoadBalancer: &capa.AWSLoadBalancerSpec{
-				Name:             ptr.To(clusterID.InfraID + "-ext"),
+				Name:             ptr.To(clusterID.InfraID + "-int"),
 				LoadBalancerType: capa.LoadBalancerTypeNLB,
-				Scheme:           &capa.ELBSchemeInternetFacing,
+				Scheme:           &capa.ELBSchemeInternal,
 				AdditionalListeners: []capa.AdditionalListenerSpec{
 					{
 						Port:     22623,
 						Protocol: capa.ELBProtocolTCP,
 					},
 				},
+				IngressRules: []capa.IngressRule{
+					{
+						Description: "MCS traffic from cluster",
+						Protocol:    capa.SecurityGroupProtocolTCP,
+						FromPort:    22623,
+						ToPort:      22623,
+						CidrBlocks:  []string{"10.0.0.0/16"},
+					},
+					{
+						Description: "Kube API traffic",
+						Protocol:    capa.SecurityGroupProtocolTCP,
+						FromPort:    6443,
+						ToPort:      6443,
+						CidrBlocks:  []string{"0.0.0.0/0"},
+					},
+				},
+			},
+			SecondaryControlPlaneLoadBalancer: &capa.AWSLoadBalancerSpec{
+				Name:             ptr.To(clusterID.InfraID + "-ext"),
+				LoadBalancerType: capa.LoadBalancerTypeNLB,
+				Scheme:           &capa.ELBSchemeInternetFacing,
+				IngressRules: []capa.IngressRule{
+					{
+						Description: "Kube API traffic",
+						Protocol:    capa.SecurityGroupProtocolTCP,
+						FromPort:    6443,
+						ToPort:      6443,
+						CidrBlocks:  []string{"0.0.0.0/0"},
+					},
+				},
```

Comment:
Are we blocking this change waiting for PR https://github.com/openshift/installer/pull/8149 ?